### DISCUSSION TITLE
How to disable http port access

### DISCUSSION THREAD

**Discussion by huang-x-h** [apache/apisix#7800] [Q&A]
When enable `ssl`, we can access by `https`, now want to close `http` port, so user can not access `http` site, in addition to use firewall forbbiden.

**Comment by tokers** [comment]
Looks like we cannot.

**Reply by huang-x-h** [reply]
😞

**Comment by Zeeeeta** [comment]
I have the same requirement, but seems there is no any option to disable http on global or route level.
I could only rely on firewall as well as setting https redirection per route?
